config/initializers/secure_headers.rb

```diff
@@ -7,7 +7,7 @@
   config.x_permitted_cross_domain_policies = 'none'
 
   connect_src = ["'self'", '*.newrelic.com', '*.nr-data.net', '*.google-analytics.com',
-                 'sentry.io']
+                 'services.assureid.net']
```

Comment:
I suppose that since we are keeping the other flow in there we need to keep sentry in the CSP here. I'll go back and change that.